test/lib/cli/upgrade_command_test.rb

```diff
@@ -230,6 +230,7 @@ def foreman_maintain_update_unavailable
       end
 
       it 'with --phase it runs only a specific phase of the upgrade' do
+        foreman_maintain_update_unavailable
```

Comment:
why was that needed?

Reply:
I'll remove it, as I have come to realize there is a intermittent test failure that happens in some runs that I will try to fix stand-alone.